.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,53 @@
+name: Benchmarks
```

Comment:
hmm.. I know we had it like this in UR, but is this workflow required? we could move workflow_dispatch to "benchmarks-reusable.yml" and that should work as well